docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Undertow.adoc

```diff
@@ -249,6 +249,52 @@ be matched
 serve up requests that do not match anything.
 |=======================================================================
 
+==== Console Access Logging
+
+Each host allows for access logging to the console which writes structured data in JSON format. This only writes to
+`stdout` and is a single line of JSON structured data.
+
+The `attributes` management model attribute is used to determine which exchange attributes should be logged. This is
+similar to the `pattern` used for traditional access logging. The main difference being since the data is structured
+the ability to use defined keys is essential.
+
+A `metadata` attribute also exists which allows extra metadata to be added to the output. The value of the attribute is
+an arbitrary key/value pair. The values can include management model expressions, which must be resolvable when the
```

Comment:
I'm not going to block merging for this as I want it in but the value is a *set* of arbitrary key/value pairs, not a single pair. The CLI example is a set.

Reply:
Good catch. I went ahead and fixed that.